janus_core/cli.py

```diff
@@ -90,7 +93,10 @@ def _parse_typer_dicts(typer_dicts: list[TyperDict]) -> list[dict]:
     for i, typer_dict in enumerate(typer_dicts):
         typer_dicts[i] = typer_dict.value if typer_dict else {}
         if not isinstance(typer_dicts[i], dict):
-            raise ValueError(f"{typer_dicts[i]} must be passed as a dictionary")
+            raise ValueError(
+                f"""{typer_dicts[i]} must be passed as a dictionary wrapped in quotes.\
+ For example, "{{'key' : value}}" """
+            )
```

Comment:
I usually prefer separating error from process if possible
```suggestion
if any(not isinstance(td.value, dict) for td in typer_dicts if td):
    raise ValueError(
                f"""Dicts must be passed as a dictionary wrapped in quotes.\
 For example, "{{'key' : value}}" """
 
typer_dicts = [typer_dict.value if typer_dict else {} for typer_dict in typer_dicts]
```

Reply:
Only problem with this solution is that it doesn't tell you which (non-)dict is causing the error.

Reply:
That is true, but could also do a find in case of failure? Could also do a filter and iterate through. I don't know. How useful is printing that "non-dict"?

Reply:
Also like I say, what's there works, and I'm happy with it

Reply:
Think I'll leave this as is just to be as explicit as possible with the cause of the error, given that I can imagine passing dictionaries incorrectly is likely to be one of the more common issues.